### PR TITLE
Add warning about using EventChannel with pausable systems

### PR DESCRIPTION
**Pull request by udoprog**
CC #1204

### CODE REVIEW

File: amethyst_core/src/system_ext.rs [hunk 1]
```diff
@@ -10,6 +10,15 @@ use specs::prelude::{Read, System};
 pub trait SystemExt {
     /// Make a system pausable by tying it to a specific value of a resource.
     ///
+    /// When the value of the resource differs from `value` the system is considered "paused",
```
Comment (TimonPost) [inline amethyst_core/src/system_ext.rs]: Nice, nitpick incomming: "paused" => 'paused'

Comment (udoprog) [reply amethyst_core/src/system_ext.rs]: Thanks, how come single quotes is correct here?

**Comment by azriel91** [comment]
bors r=torkleyy,TimonPost,azriel91

File: amethyst_core/src/system_ext.rs (hunk 1 above)
Comment (TimonPost) [reply amethyst_core/src/system_ext.rs]: Double quote is for marking a statement said by someone. If you just want to highlight a word you should use single quote.

**Pull request merged**